src/forms/signpost/SignPost.tsx

```diff
@@ -156,8 +158,13 @@ export class SignPost extends React.PureComponent<SignPostProps> {
 
     buildSignPostTriggerButton = () => {
         const {isOpen} = this.state;
-        const {openAt} = this.props;
-        return (
+        const {openAt, showCustomOpenAt} = this.props;
+        const className = isOpen ? "signpost-action signpost-trigger active" : "signpost-action signpost-trigger";
```

Comment:
Please create classNames file or constants for these classes

Reply:
created className constants.